fronts-client/src/components/form/BoostToggles.tsx

```diff
@@ -0,0 +1,109 @@
+import InputRadio from 'components/inputs/InputRadio';
+import React from 'react';
+import { Field } from 'redux-form';
+import type { BoostLevels } from 'types/Collection';
+
+/**
+ * This function generates a set of radio button toggles for selecting a card's "boost level"
+ * in a flexible collection. Boost levels help establish a visual hierarchy within a collection.
+ *
+ * ## Boost Levels:
+ * - **default**: No boost
+ * - **boost**: Slight emphasis
+ * - **megaBoost**: Stronger emphasis
+ * - **gigaBoost**: Most prominent
+ *
+ */
+
+type Toggle = { id: string; value: BoostLevels; label: string };
+
+// Define all possible boost levels for toggles
+const TOGGLES: Record<BoostLevels, Toggle> = {
+	default: {
+		id: 'boostlevel-0',
+		value: 'default',
+		label: 'Default',
+	},
+	boost: {
+		id: 'boostlevel-1',
+		value: 'boost',
+		label: 'Boost',
+	},
+	megaboost: {
+		id: 'boostlevel-2',
+		value: 'megaboost',
+		label: 'Mega Boost',
+	},
+	gigaboost: {
+		id: 'boostlevel-3',
+		value: 'gigaboost',
+		label: 'Giga Boost',
+	},
+};
+
+/**
+ * ## Presets:
+ * To simplify the boosting process for editors, we provide preset groups for different collection types.
+ * We override the labels in some groups so they make more sense in the context of the collection.
+ *
+ * ### Flexible General (4 Groups):
+ * - **Group 0 (Standard)**: Default, Boost, MegaBoost
+ * - **Group 1 (Big)**: Boost, MegaBoost
+ * - **Group 2 (Very Big)**: MegaBoost
+ * - **Group 3 (Splash)**: Default, Boost, MegaBoost, GigaBoost
+ *
+ * ### Flexible Special (2 Groups):
+ * - **Group 0 (Standard)**: Default, Boost, MegaBoost
+ * - **Group 1 (Splash)**: Default, Boost, MegaBoost, GigaBoost
```

Comment:
Discussed in person, but just a quick note here that the grouping in flexible special is... special, and we need to allow gigaboost for standard stories too (so splashes have the correct options). 

In the future we can revisit the grouping in flexible special so we have something a bit more explicit and similar to the flexible general groups though!